Other/Cockroach/Cockroach.cs

```diff
@@ -0,0 +1,831 @@
+//cs_include Scripts/ShadowsOfWar/MergeShops/TimekeepMerge.cs
+//cs_include Scripts/ShadowsOfWar/MergeShops/StreamwarMerge.cs
+//cs_include Scripts/ShadowsOfWar/MergeShops/WorldsCoreMerge.cs
+//cs_include Scripts/ShadowsOfWar/MergeShops/ManaCradleMerge.cs
+//cs_include Scripts/ShadowsOfWar/CoreSoWMats.cs
+//cs_include Scripts/Other/Classes/ArchMage/CoreArchMage.cs
+//cs_include Scripts/Farm/BuyScrolls.cs
+//cs_include Scripts/Chaos/ChaosAvengerPreReqs.cs
+//cs_include Scripts/Other/Weapons/ExaltedApotheosisPreReqs.cs
+//cs_include Scripts/Darkon/DarkonDebris2ReconstructedPrerequisites.cs
+//cs_include Scripts/ShadowsOfWar/UltraSpeakerMergePreReqs.cs
+//cs_include Scripts/Story/ElegyofMadness(Darkon)/DarkonGarden.cs
+//cs_include Scripts/Chaos/AscendedDrakathGear.cs
+//cs_include Scripts/Good/BLoD/2UltimateBlindingLightofDestiny.cs
+//cs_include Scripts/Hollowborn/CoreHollowborn.cs
+//cs_include Scripts/Hollowborn/HollowbornDoomKnight/CoreHollowbornDoomKnight.cs
+//cs_include Scripts/Hollowborn/HollowbornPaladin/CoreHollowbornPaladin.cs
+//cs_include Scripts/Other/FireAvatarFavorFarm.cs
+//cs_include Scripts/Story/Artixpointe.cs
+//cs_include Scripts/Story/ShadowsOfChaos/CoreSoC.cs
+//cs_include Scripts/Other/Classes/VerusDoomKnight.cs
+//cs_include Scripts/Other/MergeShops/TerminaTempleMerge.cs
+//cs_include Scripts/Seasonal/TalkLikeaPirateDay/DoomPirateStory.cs
+//cs_include Scripts/Seasonal/TalkLikeaPirateDay/MergeShops/DoomPirateHaulMerge.cs
+//cs_include Scripts/Nation/VHL/CoreVHL.cs
+//cs_include Scripts/Nation/AssistingCragAndBamboozle[Mem].cs
+#endregion
+
+
+
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Models.Shops;
+using Skua.Core.Options;
+using Skua.Core.Utils;
+
+public class Cockroach
+{
+    public static IScriptInterface Bot => IScriptInterface.Instance;
+    public static CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new();
+    public CoreAdvanced Adv = new();
+
+    public BladeOfAwe BoA = new(); 
+    
+    public CoreBLOD BLOD = new();
+
+    public UnlockForgeEnhancements forgeUnlocks = new();
+    public CoreSepulchure CoreSS = new();
+    public CoreYnR YNR = new();
+
+    public YokaiQuests Yokai = new();
+
+    public ArchDoomKnight ADK = new();
+
+    public Awescended awescended = new();
+
+    public ChaosAvengerClass CaV = new();
+
+    public ExaltedApotheosisPreReqs EA = new();
+
+    public PrinceDarkonsPoleaxePreReqs PDPP = new();
+    
+    public DarkonDebris2ReconstructedPrerequisites DDRP = new();
+
+    public UltraSpeakerMergePreReqs US = new();
+
+    public MalgorsArmorSet MAS = new();
+
+    public CoreNSOD NSOD = new();
+
+    public CoreDailies Daily = new();
+
+    public BuyScrolls Scroll = new();
+//Classes
+    public DragonOfTime DoT = new();
+    public ArchPaladin AP = new();
+    public CoreLR LR = new();
+    public CoreArchMage AM = new();
+    public VerusDoomKnightClassPrep VDK = new();

```

Comment:
no longer exists, as bogi put the whole thing together, dont add this as it requires ultras.